Index: step/autotests/test_metaobject.cc
===================================================================
--- step.orig/autotests/test_metaobject.cc	2015-09-17 20:36:29.329219066 +0200
+++ step/autotests/test_metaobject.cc	2015-10-29 18:36:32.605132864 +0100
@@ -170,7 +170,7 @@
     QCOMPARE( testObject->property1(), 1.1 );
     QVERIFY( property->writeVariant(object, QVariant(2.2)) );
     QCOMPARE( testObject->property1(), 2.2 );
-    QCOMPARE( property->readString(object), QString("2.2") );
+    QCOMPARE( QString::number(property->readVariant(object).value<double>()), QString("2.2") );
     QCOMPARE( property->readVariant(object), QVariant(2.2) );
 
     QVERIFY( !property->writeString(object, "not number") );
